app/views/reviews/_review.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="review row">
+  <div class="columns small-12 medium-6 medium-centered">
+    <div class="row">
+      <%= render partial: 'votes/vote_buttons', locals: { review: review } %>
+      <div class="rating columns small-2">
+        <%= review.rating %>
+      </div>
+      <div class="body columns small-9">
+        <%= review.body %>
+      </div>
+      <% if current_user %>
+        <%= button_to 'Edit', edit_venue_review_path(@venue, review), 
```

Comment:
Should this be checking to see if current_user and current_user.id = review.user_id  to make sure that only the person who wrote the review can edit it? 
(this should also incorporate the admin being able to edit it, we can refactor later)